packages/webapp/src/components/Crop/BedPlan/styles.module.scss

```diff
@@ -10,6 +10,6 @@
 }
 
 
-.padding-bottom-40 {
+.paddingBottom40 {
```

Comment:
wouldn't you might as well make this an inline style if you're going to name the class by the style itself?

Reply:
this applies to all other appearances of this classname